Draggable/explainer.md

```diff
@@ -0,0 +1,165 @@
+# Draggable App Region 
+
+## Authors
+
+- [Diego Gonzalez](https://github.com/diekus)
+
+## Participate
+
+- [Issue tracker](https://github.com/MicrosoftEdge/MSEdgeExplainers/labels/Draggable%20App%20Region)
+
+## Status of this Document
+
+This document is intended as a starting point for engaging the community and
+standards bodies in developing collaborative solutions fit for standardization.
+As the solutions to problems described in this document progress along the
+standards-track, we will retain this document as an archive and use this section
+to keep the community up-to-date with the most current standards venue and
+content location of future work and discussions.
+
+* This document status: **Active**
+* Expected venue: [CSS Working Group](https://www.w3.org/Style/CSS/)
+* Current version: this document
+
+## Table of Contents
+
+- [Draggable App Region](#draggable-app-region)
+  - [Authors](#authors)
+  - [Introduction](#introduction)
+  - [Goals](#goals)
+  - [Non Goals](#non-goals)
+  - [Feature Usage](#feature-usage)
+  - [Property Values](#property-values)
+  - [Current Property Behaviour](#current-property-behaviour)
+  - [Sample Use Case](#sample-use-case)
+  - [Stakeholder Feedback / Opposition](#stakeholder-feedback--opposition)
+  - [Accessibility, Privacy, and Security Considerations](#accessibility-privacy-and-security-considerations)
+  - [References \& acknowledgements](#references--acknowledgements)
+
+
+## Introduction
+
+The `-webkit-app-region` and `app-region` properties exist as a styling feature used primarily in desktop web applications to define draggable regions of a window. This is useful when applications recreate a custom title bar or title bar region, as it **allows users to click and drag the window to move it when the default OS window chrome is not present**.
+
+This property is used by several technologies to move a window in the OS, including:
+* [Web Apps through the WCO API](https://wicg.github.io/window-controls-overlay/)
+* [Electron](https://www.electronjs.org/docs/latest/tutorial/custom-window-interactions#custom-draggable-regions)
+
+The property is not currently standard, this leaves a gap on implementations that needs to be fixed. 
+
+## Goals
+
+* Standardize the `app-region` CSS property.
+* Allow a _standalone_ window to be moved by a user when no other platform UX is present.
```

Comment:
nit - it took me a minute to understand what this means and might confuse other people. Suggest being more crispy with wording. Something like "Allowing _standalone_ app window to define draggable and non-draggable region and support scenarios such as custom title bar"